packages/manager/src/features/TopMenu/TopMenu.tsx

```diff
@@ -41,7 +43,7 @@ export const TopMenu = React.memo((props: TopMenuProps) => {
     <React.Fragment>
       {loggedInAsCustomer && (
         <Box bgcolor="pink" padding="1em" textAlign="center">
-          <Typography color="black" fontSize="1.2em">
+          <Typography color={theme.colorTokens.Neutrals.Black} fontSize="1.2em">
```

Comment:
We can do this to eliminate the need to import `useTheme`:

```suggestion
          <Typography
            color={(theme) => theme.colorTokens.Neutrals.Black}
            fontSize="1.2em"
          >
```